app/controllers/users_controller.rb

```diff
@@ -290,7 +243,8 @@ def auth_success
           failed_login t("sessions.new.auth failure")
         end
       else
-        redirect_to :action => "new", :nickname => name, :email => email,
+        verified_email = UsersController.message_hmac(email) if email_verified && email
```

Comment:
`verified_email` is no longer a email address and it's named `email_hmac` elsewhere. Should it be renamed here too?